docs/source/applications/avalon/avalon_importing_entities.rst

```diff
@@ -0,0 +1,56 @@
+===============
+Importing media into Avalon
+===============
```

Comment:
Since this hasn't been merged yet, make the `=` align with the last `n`.  This gets rid of a warning when you build sphinx.